src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +70,65 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
```

Comment:
@andreasohlund - i am going to make the web api call only for system.interactive and in debugging mode. This will avoid executions on server deployments. Stay tuned. Will ping you when the changes are incorporated. I've been thinking about this as well and @bording also pinged me about this.